Dockerfile

```diff
@@ -0,0 +1,15 @@
+# docker build . -t sncli
+# docker run --rm -it -v /tmp:/tmp -v "$HOME/.sncli/:/root/.sncli/" -v "$HOME/.snclirc:/root/.snclirc" sncli
+FROM python:3.8-buster
```

Comment:
Can this be a more generic? `python:3` should be fine, and pull the latest version at the time.

Reply:
With docker I always try to make things more reproducible and not just pull latest versions

Reply:
Ok, fair enough. :+1: